automation/Extension/firefox/data/create_http_requests_table.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS http_requests_ext(
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  crawl_id INTEGER NOT NULL,
+  visit_id INTEGER NOT NULL,
+  url VARCHAR(500) NOT NULL,
```

Comment:
This was done out of habit. I'll update these to `TEXT` since that's what sqlite uses anyway.